tests/drivers/regulator/voltage/boards/rpi_pico.overlay

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2023 TOKITA Hiroshi <tokita.hiroshi@fujitsu.com>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/adc/adc.h>
+
+/ {
+	resources: resources {
+		compatible = "test-regulator-voltage";
+		regulators = <&vreg_core>;
+		io-channels = <&adc 0>;
```

Comment:
maybe add a comment on how to wire this up (seems you need to solder a wire)

Reply:
I can't attach a picture here, so I described it in the comments.